library/core/src/ffi/mod.rs

```diff
@@ -599,3 +599,13 @@ extern "rust-intrinsic" {
     #[rustc_nounwind]
     fn va_arg<T: sealed_trait::VaArgSafe>(ap: &mut VaListImpl<'_>) -> T;
 }
+
+// Link the MSVC default lib
+#[cfg(all(windows, target_env = "msvc"))]
+#[link(
+    name = "/defaultlib:msvcrt",
```

Comment:
Is it still possible to use ucrt instead of msvcrt with this change?

Reply:
"msvcrt" is the name of the .lib file, not the dll to be linked. For compatibility reasons, it'll likely never change that name. It does however change content and so the UCRT is always used on recent Visual Studio versions. `/nodefaultlib:msvcrt` can be used to remove it (or `/nodefaultlib` alone can be used to remove all import libraries)